scripts/mkcloud

```diff
@@ -1460,6 +1460,7 @@ sleep 2
 for cmd in `echo $steplist` ; do
     echo
     echo "============> MKCLOUD STEP START <============"
+    echo "$cmd started at `date '+%Y-%m-%d %H:%M:%S'`"
```

Comment:
1.
The beginning of the line differs, this makes makes grepping more complex
Could agree on a fixed prefix for this log line? Something like:

```
echo "step started at `date '+%Y-%m-%d %H:%M:%S'`: $cmd"
```

2.
Please move it one line below, so after the line "MKCLOUD step:..". This is our marker for automatic parsing. The "====" is only the visual separator.